src/pages/RequestInvite/index.tsx

```diff
@@ -254,56 +271,49 @@ export class RequestInvite extends React.Component<Props, State> {
             }}
           />
         </section>
-        {this.props.notSignedIn ? (
-          this.renderLogIn()
-        ) : this.props.isAuthLoaded ? (
+        {this.props.isLoading ? (
+          <Loading />
+        ) : this.props.loggedInFirebaseUser ? (
           this.renderForm()
         ) : (
-              <Loading />
-            )}
+          this.renderLogIn()
+        )}
       </RequestInviteElem>
     );
   }
 }
 
 function mapStateToProps(state: AppState, ownProps: OwnProps): Partial<Props> {
-  const memberId = ownProps.match.params.memberId;
   const isAuthLoaded = state.auth.isLoaded;
-  const toMemberDoc = getMemberDocByMid(state, memberId);
-
-  const stateToPropsMap = {
-    isAuthLoaded,
-    memberId,
-    isToMemberDocLoaded: toMemberDoc && !toMemberDoc.isFetching,
-    toMemberDoc,
-    notSignedIn: true
-  };
+  const loggedInFirebaseUser = state.auth.firebaseUser;
+  const loggedInMember: Member | undefined = loggedInFirebaseUser
+    ? state.membersNew.byUid[loggedInFirebaseUser.uid]
+    : undefined;
+  const loggedInFullName: string | undefined | null = loggedInMember
```

Comment:
why can this be both `undefined` and `null`? Looks like just `undefined`

Reply:
Same issue as mentioned in #111, we have inconsistent typing around `null` and `undefined`, so I think this is a little messy. Will leave another TODO to improve this in a later PR.